source/templates.tex

```diff
@@ -1829,7 +1829,7 @@
 the normal form\iref{temp.constr.normal} of the concept-id
 after substitution of \tcode{C}.
 \begin{note}
-Normalization of \tcode{CI} might be ill-formed; no diagnostics is required.
+Normalization of \tcode{CI} might be ill-formed with no diagnostic required.
```

Comment:
Uhhhh, "might" in a note.  Peril awaits.

Please replace with "can" as a drive-by fix.